Tests/EntitlementsSpec.swift

```diff
@@ -0,0 +1,394 @@
+import Quick
+import Nimble
+import KindeSDK
+import Foundation
+
+class EntitlementsSpec: QuickSpec {
```

Comment:
The tests for `fetchEntitlements` and `getAllEntitlements` have async functions and require using the `AsyncSpec` to be able to run the tests without a compilation error. 


```suggestion
class EntitlementsSpec: AsyncSpec {
```